### DISCUSSION TITLE
How i can get a value from array in primereact datatable ?

### DISCUSSION THREAD

**Discussion by peruccii** [primefaces/community#951] [PrimeReact]
export interface ConsultantInterface {
  id: number,
  email: string,
  nome: string,
  cpf: string,
  taxa_hora: number,
  rg: string,
  linkedin: string,
  status: { id: number, nome: string},
  endereco: {
    cep: string;
    cidade: string;
    bairro: string;
    complemento: string;
    uf: string;
    logradouro: string;
    numero: string;
    pais: string;
}; 
  idioma:  [
    {
      id: number,
      nome: string,
      niveis: [{id: number, nome: string}]
    }
  ]
  especialidade: Array<{
    id: number,
    nome: string,
    niveis: Array<{ id: number, nome: string }>,
  }>,
  modelo_trabalho: Array<{ id: number, nome: string }>,
  regime_trabalho: Array<{ id: number, nome: string }>,
  data_nascimento: string,
  sexo: {id: number, nome: string},
  fonte_recrutamento: { id: number, nome: string },
  telefone: {
    ddd: { ddd: number, id: number },
    ddi: { ddi: string, id: number },
    numero: string,
  },
  telefoneSecundario: {
    ddd: { ddd: number, id: number },
    ddi: { ddi: string, id: number },
    numero: string,
  }
  alocacao?: Array<string>;
}

 const [filters, setFilters] = useState({
    nome: { value: null , matchMode: FilterMatchMode.CONTAINS },
    'endereco.estado': { value: valorEstado , matchMode: FilterMatchMode.CONTAINS },
    'endereco.cidade': { value: null, matchMode: FilterMatchMode.STARTS_WITH },
    cidade: { value: null, matchMode: FilterMatchMode.STARTS_WITH },
    taxa_hora: { value: null, matchMode: FilterMatchMode.LESS_THAN },
    representative: { value: null, matchMode: FilterMatchMode.IN },
    'status.nome': { value: null, matchMode: FilterMatchMode.EQUALS },
    idioma: { value: valorIdioma, matchMode: FilterMatchMode.EQUALS},
    
  });
  
  How can i get the value of the language when it is an array ?
  When its a object i can do this: 'endereco.estado', but in array, example, 'idioma.nome' does not work
  
  can someone helpe me

**Comment by mertsincan** [comment]
Sorry, but I didn't understand it. If you want to show object values in columns, you can use `field="idioma[0].nome"`. For now, closed. If the problem persists, could you please create a Github issue with a sample stackblitz link to examine in detail?
